### PR TITLE
Add org.hectabit.PageBurger

### PR DESCRIPTION
**Pull request by Arzumify**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: org.hectabit.PageBurger.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "make install DESTDIR=/app/bin/ SHAREDIR=/app/share/"
+      ]
+    }
+  ],
+  "finish-args": [
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=ipc"
+  ]
```
Comment (bbhtt) [inline org.hectabit.PageBurger.json:29]: Put this on top before `modules`

File: org.hectabit.PageBurger.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "id": "org.hectabit.PageBurger",
```
Comment (bbhtt) [inline org.hectabit.PageBurger.json:2]: Do you own hectabit.org?

Comment (Arzumify) [reply org.hectabit.PageBurger.json:2]: > Do you own hectabit.org?

Yes

**Comment by Arzumify** [comment]
bot, build org.hectabit.PageBurger

**Comment by Arzumify** [comment]
Strange, the Appstream builds fine locally

**Comment by Arzumify** [comment]
Strange, builds fine locally

**Comment by Arzumify** [comment]
bot, build org.hectabit.PageBurger

File: org.hectabit.PageBurger.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        {
+          "type": "git",
+          "url": "https://centrifuge.hectabit.org/HectaBit/PageBurger-App.git",
+          "tag": "1.0",
+          "commit": "26fae71110cea6568e5c86b6746dbd1a303bcdb7"
+        }
+      ],
+      build-commands": [
+        "make install DESTDIR=/app/bin/ SHAREDIR=/app/share/"
+      ]
```
Comment (bbhtt) [inline org.hectabit.PageBurger.json:27]:
```suggestion
      "build-commands": [
        "make install DESTDIR=/app/bin/ SHAREDIR=/app/share/"
      ]
```

This should be before sources

File: org.hectabit.PageBurger.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://centrifuge.hectabit.org/HectaBit/PageBurger-App.git",
+          "tag": "1.0",
+          "commit": "26fae71110cea6568e5c86b6746dbd1a303bcdb7"
+        }
+      ],
+      build-commands": [
```
Comment (hfiguiere) [inline org.hectabit.PageBurger.json:25]: it's missing a `"`. hence the failure.

Comment (Arzumify) [reply org.hectabit.PageBurger.json:25]: Whoops

**Comment by Arzumify** [comment]
bot, build org.hectabit.PageBurger

**Comment by Arzumify** [comment]
bot, build org.hectabit.PageBurger

File: org.hectabit.PageBurger.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
+{
+  "id": "org.hectabit.PageBurger",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "pageburger",
+  "finish-args": [
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
```
Comment (bbhtt) [inline org.hectabit.PageBurger.json:10]: Add `--device=dri`, shows a bunch of mesa related warnings.

Comment (Arzumify) [reply org.hectabit.PageBurger.json:10]: ok then

**Comment by Arzumify** [comment]
bot, build org.hectabit.PageBurger

**Pull request closed without merging**